src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-MLPERF.json

```diff
@@ -29,31 +29,10 @@
         {
             "Type": "MLPerfExecutor",
             "Parameters": {
-                "Scenario": "inferencemodels-rnnt",
-                "Model": "rnnt",
+                "Scenario": "inferencemodels-3d-unet",
```

Comment:
We seem to be regressing the number of inferencing models we are running here. The MLPerf workload is not worth it if we cannot run the suite of models.

Reply:
The suite set is different in 4.1 vs 2.0. I asked Sai to first get to a working state first with the 2 benchmark that works by default

This is the new set for 4.1 and rnnt is not here.
![image](https://github.com/user-attachments/assets/51a018ad-ae50-4097-a59a-b9ae60b38347)